test/checkpoint/test_checkpoint.c

```diff
@@ -261,9 +261,12 @@ wt_connect(const char *config_open)
     static WT_EVENT_HANDLER event_handler = {
       handle_error, handle_message, NULL, NULL /* Close handler. */
     };
+    WT_RAND_STATE rnd;
     int ret;
     char config[512];
     char timing_stress_config[512];
+    bool fast_eviction_target = false;
```

Comment:
Place the initialisation below. WiredTiger likes to place the initialisation of variables after the declaration phase.